### DISCUSSION TITLE
e-mails to bookers by organisers

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#362] [Ideas]
Hi,
I'm looking for a simple way for event organisers to send an email to their event'bookers. I don't want to give them access to "Email management" in the backend. The organisers use EMEFS to schedule their events. An equivalent plugin for sending their emails would be perfect.
Have a nice day

**Comment by liedekef** [comment]
sorry, but currently EME doesn't offer this as functionality

**Reply by f14m07** [reply]
Yes, but is there a possibility with a hook and snippet, have you a piece of advice for me? Can I restrict access to the email section for bookers only?

**Reply by liedekef** [reply]
See the EME settings, tab Access. There you have the "Email" section, where you can set the needed capabilities someone needs to have to send emails for his/her own events (the logged-in user is either contact person or author of the event).

**Reply by f14m07** [reply]
Yes I saw but he has also access to other tab 'Generic emails', 'mailings', 'sent emails' and not only 'Event related email'. So he can see also sent emails about other events or subjects.

**Reply by liedekef** [reply]
The next version will have a little more granular permissions for mail, that should help here.

**Reply by f14m07** [reply]
Thanks a lot for this good news
